### ISSUE TITLE
Initial Installation/Connection Problems.

### ISSUE THREAD

**Issue by kopepasah** [Varying-Vagrant-Vagrants/VVV#404]
When installing and running `vagrant up` for the first time I installations always stops completely when trying to download the various installations of WordPress ([see here](https://gist.github.com/kopepasah/25796ab2c759e76c0974#file-0-connection-error-L3-L7) as an example).

When I run `vagrant provision` after the process stops, the next download installs; however it stops again ([on the next install](https://gist.github.com/kopepasah/25796ab2c759e76c0974#file-1-connection-error-L2-L6)) once the download is complete.

No matter how many times I run vagrant provision, the install never seems to complete. In addition, there is always this [consistent message](https://gist.github.com/kopepasah/25796ab2c759e76c0974#file-3-consistent-error-L2-L14) at the end of my installs (unsure if it is related).

Any help would be appreciated.

__System Info__
```
OSX 10.9.4
VBox 4.3.12
Vagrant 1.6.3
```

**Comment by jeremyfelt** [comment]
Hey @kopepasah, sorry you never got a response to this. :disappointed: 

Hopefully everything worked out. If not, please feel free to open another issue or reopen this one.

**Comment by kopepasah** [comment]
@jeremyfelt no worries. Seems that it was a temporary connection issue (most likely on my end). Everything is working fine now.
